orchestrator/careplancontributor/service.go

```diff
@@ -438,31 +442,22 @@ func (s Service) authorizeScpMember(request *http.Request) (*ScpValidationResult
 		return nil, err
 	}
 
-	// Use extract CarePlan ID to be used for our query that will get the CarePlan and CareTeam in a bundle
-	var bundle fhir.Bundle
-	err = cpsFHIRClient.Search("CarePlan", url.Values{"_id": {carePlanId}, "_include": {"CarePlan:care-team"}}, &bundle)
+	var carePlan fhir.CarePlan
+
+	err = cpsFHIRClient.ReadWithContext(request.Context(), "CarePlan/"+carePlanId, &carePlan)
```

Comment:
```suggestion
	err = cpsFHIRClient.ReadWithContext(request.Context(), carePlanRef, &carePlan)
```

The ID parsing can then be removed, probably